requirements.txt

```diff
@@ -1,8 +1,8 @@
-SQLAlchemy==1.3.23
-Faker==6.6.0
+SQLAlchemy==1.4.3
```

Comment:
We can't upgrade to sqlalchemy 1.4 yet, it has more significant changes that we need to take a look at.